rudder-web/src/main/scala/com/normation/rudder/web/components/ShowNodeDetailsFromNode.scala

```diff
@@ -98,6 +98,17 @@ class ShowNodeDetailsFromNode(
   private[this] val nodeRepo             = RudderConfig.woNodeRepository
   private[this] val asyncDeploymentAgent = RudderConfig.asyncDeploymentAgent
   private[this] val configService = RudderConfig.configService
+  private[this] var nodeInfo = nodeInfoService.getNodeInfo(nodeId) match {
+    case Full(Some(info)) => info
+    case eb : EmptyBox =>
+      val e = eb ?~! s"Could not get Node '${nodeId.value}' details"
+      logger.error(e.messageChain)
+      throw new Exception(e.messageChain)
```

Comment:
i'm really not convinced by throwing an exception when loading the page. 
With this, we go from a safe-ish approach where the error The node with id '${nodeId.value}' was not found was displayed to user to an exception, that won't probably be of any help to user :/